### PR TITLE
Fix pod sharing for utsmode

### PR DESCRIPTION
**Pull request by rhatdan**
We should be sharing cgroups namespace by default in pods
uts namespace sharing was broken in pods.

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by mheon** [comment]
Does this disable pod UTS namespace sharing?

**Comment by rhatdan** [comment]
No this should allow a user to specify --uts container:UUID so you can share uts between containers.

**Comment by rhatdan** [comment]
bot, retest this please

**Comment by rhatdan** [comment]
@giuseppe Any idea why the rootless tests keep timing out?

File: pkg/spec/createconfig.go [hunk 1]
```diff
@@ -421,8 +421,13 @@ func (c *CreateConfig) GetContainerCreateOptions(runtime *libpod.Runtime) ([]lib
 		options = append(options, libpod.WithIPCNSFromPod(pod))
 	}
 
-	if IsPod(string(c.UtsMode)) {
```
Comment (mheon) [inline pkg/spec/createconfig.go]: Did we lose the IsPod here?

**Comment by rhatdan** [comment]
@mheon @giuseppe @umohnani8 @baude @vrothberg PTAL

**Comment by mheon** [comment]
LGTM
@TomSweeneyRedHat @baude @umohnani8 PTAL

**Comment by baude** [comment]
LGTM

**Comment by mheon** [comment]
@rh-atomic-bot r+

**Pull request closed without merging**